### ISSUE TITLE
ARM64 Support for Docker Images

### ISSUE THREAD

**Issue by odidev** [ibmruntimes/ci.docker#106]
Hi Team,

I tried to build the docker image of this package for arm64 and found that it is calling [index.yml](https://github.com/ibmruntimes/ci.docker/blob/master/ibmjava/8/sfj/alpine/Dockerfile#L52-#L77) file for different architecture which is further using some ibm runtime binaries which are not available for arm64.

I have looked for the index.yml and found this [link ](https://public.dhe.ibm.com/ibmdl/export/pub/systems/cloud/runtimes/java/meta/sfj/linux/)which shows that index.yml is not available for arm64.

I also tried to get the source code of these index.yml files and their binaries but I was unable to find one.

I have also checked the [ibmjava image](https://hub.docker.com/_/ibmjava) and found that it does not have the arm64 compatible tag.

Do you have any plans to release arm64 images?

I am happy to help in adding the arm64 support, but it requires the above mentioned arm64 binary or the source code which generates these index.yml files and binaries.

**Comment by pshipton** [comment]
There is no IBM ARM64 binary available at this time. https://hub.docker.com/_/adoptopenjdk is available. There are builds using Eclipse OpenJ9, which is also used by IBM Java.

**Comment by maulik-modi** [comment]
@odidev and @pshipton , I am just thinking loudly, can https://developer.ibm.com/languages/java/semeru-runtimes/downloads/ ARM64 JDK/JRE be useful here?

**Comment by pshipton** [comment]
@maulik-modi Yes, aarch64/arm64 has been available since the 0.29 release (Oct 2021), see https://hub.docker.com/_/ibm-semeru-runtimes

**Comment by sdehors-ibm** [comment]
So is there a plan to deliver an aarch64 docker image ?
